src/GeneralStateTestsFiller/stTransactionTest/CreateTransactionInvalidEOF1Filler.yml

```diff
@@ -265,7 +415,22 @@ CreateTransactionInvalidEOF1:
       - ':raw 0xef000101000d02001100601738038060176000396000f3ef000101000304000400600000aabbccdd'
       # deployed code: ef0001010003ff000400600000aabbccdd
       - ':raw 0xef000101000d02001100601738038060176000396000f3ef0001010003ff000400600000aabbccdd'
-
+      # EIP-3670 - Valid EOF init code returning invalid EOF: Truncated PUSH data at the end
+      # code: { sstore(0x1, 0x1) sstore(0x2,0x2) push2(0x1,) } - ef000101000c00600160015560026002556100
+      # init code: { mstore(0x0, 0xef000101000c0060016001556002600255610000000000000000000000000000) return(0x00, 0x13) }
+      - ':label valid_init_invalid_code_truncated_push_data :raw 0xef0001010029007fef000101000c006001600155600260025561000000000000000000000000000060005260136000f3'
+      # EIP-3670 - Valid EOF init code returning invalid EOF: Containing undefined instruction
+      # code: { sstore(0x1, 0x1) sstore(0x2,0x2)  0xf6 } -       ef000101000b0060016001556002600255f6
+      # init code: { mstore(0x0, 0xef000101000b0060016001556002600255f60000000000000000000000000000) return (0x00, 0x12) }
+      - ':label valid_init_invalid_code_undef_instr :raw 0xef0001010029007fef000101000b0060016001556002600255f6000000000000000000000000000060005260126000f3'
+      # EIP-3670 - Valid EOF init code returning invalid EOF: Containing undefined instruction after STOP
+      # code: { sstore(0x1, 0x1) sstore(0x2,0x2) stop() 0xf6 } - ef000101000c006001600155600260025500f6
+      # init code: { mstore(0x0, 0xef000101000c006001600155600260025500f600000000000000000000000000) return (0x00, 0x13) }
+      - ':label valid_init_invalid_code_undef_instr_after_stop :raw 0xef0001010029007fef000101000c006001600155600260025500f60000000000000000000000000060005260136000f3'
+      # EIP-3670 - Valid EOF init code returning invalid EOF: Containing non terminating instruction at the end
+      # code: { sstore(0x1, 0x1) sstore(0x2,0x2) stop() sstore(0x1, 0x1) }  - ef00010100100060016001556002600255006001600155
+      # init code: { mstore(0x0, 0xef00010100100060016001556002600255006001600155000000000000000000) return (0x00, 0x17) }
+      - ':label valid_init_invalid_code_nonterm_at_end :raw 0xef0001010029007fef0001010010006001600155600260025500600160015500000000000000000060005260176000f3'
```

Comment:
Also we need to test all possible non-terminating instructions in the init code I think.